Wikipedia/Code/WMFSearchViewController.m

```diff
@@ -434,7 +440,7 @@ - (void)updateSearchSuggestion:(NSString*)searchSuggestion {
 }
 
 - (CGFloat)searchFieldHeightForCurrentTraitCollection {
-    return self.traitCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact ? 44 : 64;
+    return self.traitCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact ? 44 : 72;
```

Comment:
@nirzar 3 comments:

1. You only changed the vertical size, not the size when in horizontal and the status bar disappears - is that intentional?

2. This is now the only top bar in the whole app that is taller than the iOS standard 64 - so it looks out of place. Why do we need this bar to be bigger? 

3. We now have less vertical space for the results to show above the keyboard - not so bad on the iPhone 6+ but its getting cramped on 4 inch screens.